docs/using/event-types.md

```diff
@@ -44,6 +44,69 @@ The custom schema for an event type can be as simple as
 `{ "\additionalProperties\": true }` to allow arbitrary JSON, but will usually
 have a more specific definition :)
 
+### Compatibility modes
+
+Event type schemas evolve through time and different data integration
+patterns require different guarantees when it comes to what changes
+should or should not be allowed.
+
+To accomodate these different needs Nakadi provides three distinct
+compatibility modes.
+
+#### forward
+
+Clients using older schemas can safely read events validated by newer
+schemas. It's not safe for a client using the latest schema to read
+events generated by older schemas, in the sense that there might be
+type inconsistencies due to additionalProperties.
+
+The complete set of json-schema is supported, including
+additionalProperties. Consumers willing to read events in a safe
+manner should ignore fields that are not declared in the
+schema. Making assumptions about non declared properties might lead to
+breaking changes.
+
+#### compatible
+
+This is the most strict mode. It guarantees that events published
+under this mode can be safely read using both newer and older schema
+versions. E.g. it's both forward compatible and backwards
+compatible. In other words:
+
+1. Clients using old schemas can safely read events validated by newer
+   schemas.
+2. Clients using the newest schema can safely read events validated by
+   older schemas.
+
+In order to achieve that, only a subset of json schema will be allowed
+to make sure no accidental incompatibilities or ambiguities will be
+introduced. Only compatible changes aregoing to be allowed when
+changing the schema. This mode will be required to have data stored
+and processed by the data lake.
```

Comment:
`to have data stored and processed by the data lake` -> `for the data lake to store and process data` (removing passive form)